tianshou/policy/modelfree/dqn.py

```diff
@@ -210,6 +210,8 @@ def forward(
         return cast(ModelOutputBatchProtocol, result)
 
     def learn(self, batch: RolloutBatchProtocol, *args: Any, **kwargs: Any) -> TDQNTrainingStats:
+        # set policy in train mode
```

Comment:
I would remove all of these comments, because this is clear from the docstring of `train`.